$OpenBSD: patch-Makefile,v 1.3 2011/11/14 13:48:27 sthen Exp $

Don't copy the bootstrap files for every build step; when it's done during
fake (as root) no subsequent steps (regress/clean) will succeed.

--- Makefile.orig	Sat Nov 12 19:55:03 2011
+++ Makefile	Sat Nov 12 19:56:38 2011
@@ -80,6 +80,11 @@ bootstrap: setup bootstrap/NAnt.exe bootstrap/NAnt.Cor
 	
 
 setup:
+	if [ ! -d bootstrap ]; then \
+		mkdir -p bootstrap; cp -R lib/ bootstrap/lib; cp lib/common/neutral/log4net.dll bootstrap; cp src/NAnt.Console/App.config bootstrap/NAnt.exe.config; \
+	fi
+
+xx:
 	mkdir -p bootstrap
 	cp -R lib/ bootstrap/lib
 	# Mono loads log4net before privatebinpath is set-up, so we need this in the same directory
